### PR TITLE
Dyadic Trade Dataset

### PR DESCRIPTION
**Pull request by Lilyyyy54**
Add Dyadic Trade Dataset to the Package

### CODE REVIEW

**Comment by RandiLGarcia** [comment]
Hi Lily, for some reason it's not letting me complete your pull request.

File: DIADR.Rproj [hunk 1]
```diff
@@ -0,0 +1,18 @@
+Version: 1.0
```
Comment (beanumber) [inline DIADR.Rproj]: @Lilyyyy54 please delete this file and commit the deletion. This shouldn't be here. 

Please also delete `.RData` and `dyadic_trade.RData`.

File: R/dyadic_trade-data.R [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+#'   \item{flow1}{Imports of Country A from Country B in current US millions of dollars}
+#'   \item{flow2}{Imports of Country B from Country A in current US millions of dollars}
+#'   \item{source1}{Source of data for flow1 variable (see table below)}
+#'   \item{source2}{Source of data for flow2 variable (see table below)}
+#' }
+#' 
+#' @keywords datasets
+#' 
+#' @references Correlates of War Version 3.0
+#' (\htef{http://www.correlatesofwar.org/data-sets/bilateral-trade})
```
Comment (beanumber) [inline R/dyadic_trade-data.R]: You want `\url{}` here (not `\htef{}`)

Comment (beanumber) [reply R/dyadic_trade-data.R]: Please also add yourself to `DESCRIPTION` as a `ctb`.

**Review by beanumber — CHANGES_REQUESTED** [review]

**Comment by beanumber** [comment]
@Lilyyyy54 please also delete `.RData` and add yourself to `DESCRIPTION` as a `ctb`.

**Comment by beanumber** [comment]
@RandiLGarcia this looks good to merge, but there is a conflict causing by the `.RData` file that was part of #21 . I think you can resolve the conflict by deleting `.RData` manually on your end, and then merging the pull request. Let me know if I can help you.

**Comment by RandiLGarcia** [comment]
@beanumber I need help. After I pulled I see carol's alliance data but not the .RData file in question.

**Comment by beanumber** [comment]
OK, I can pay you a visit this afternoon.

**Pull request merged**